docs/agents/sematext-agent/starting-stopping.md

```diff
@@ -0,0 +1,96 @@
+title: Restarting/stopping the Sematext Agent
+description: Info about how to restart/stop the Sematext Agent
+
+Since Sematext Agent can be [installed](installation) in different environments, there are different ways in which it can be restarted
+or stopped.
+
+## Linux
+
+To stop the agent, run:
+
+```
+sudo service sematext-agent stop
+```
+
+To restart the agent, run:
+
+```
+sudo service sematext-agent restart
+```
+
+To remove the agent, run distribution-specific command to remove the sematext-agent rpm or deb package.
+
+## Docker
+
+In Docker environment agent can be started using `docker run` command or using a docker-compose config. 
+
+To stop the agent, run:
+
+```
+docker stop st-agent
+```
+
+and remove it with:
+
+```
+docker rm st-agent
+```
+
+To avoid agent starting after `docker-compose up` is executed the next time, ensure its definition is removed from your
+docker-compose config file.
+
+## Swarm
+
+In Swarm environment agent can be started using `docker service create` command or using a docker-compose config.
+
+To remove the agent run:
+
+```
+docker service scale st-agent=0
```

Comment:
I think we could omit this command and only mention `service rm`.

Reply:
What if we say it can be used to "stop" the agent (not "remove")? Still omit?

Reply:
I tend to think that in clustered setups "stopping" the agent is rarely used. One would rather remove the service and recreate it again. If you think we should still keep mentioning both commands, we can do that.

Reply:
OK, will keep just `service rm`.